### PR TITLE
OKTA-452468 Update procedures and scope table in Implement OAuth guide

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updated the API Security > [Implement OAuth for Okta guide](https://developer.okta.com/docs/guides/implement-oauth-for-okta/main/) with revised procedures using the Admin Console and Postman to run this basic tutorial. Additionally, updated the following table: [Scopes and supported endpoints](https://developer.okta.com/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints) to include the latest available scopes visible in the Admin Console for both Classic and OIE; added the OIE tag to highlight scopes only available for OIE.

- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-452468](https://oktainc.atlassian.net/browse/OKTA-452468)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md [hunk 1]
```diff
@@ -43,17 +43,18 @@ Create the client application that you want to use with the Okta APIs.
...
+1. On the **Create a new app integration** page, select **OIDC - OpenID Connect** as the **Sign-in method**. Choose any **Application type**. Creating a web, single-page, or native app is an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token. Click **Next**.
 
     > **Note:** It is important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, and would break the sign-in or sign-out flow.
 
-1. Enter a name for your app integration. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns a browser (along with the token) after the user finishes authenticating.
-1. We recommend that you always use the Authorization Code grant flow and clear the **Implicit (Hybrid)** check box.
+1. Enter a name for your app integration. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns a browser (along with the token) after the user finishes authenticating. You can use the default URI for this exercise.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md:50]: In the UI, **Grant type** section appears before the **Sign-in redirect URIs** section. Not a major snag, but thinking like a new user and seeing them in a slightly different order threw me a bit.

**Review by brentschaus-okta — APPROVED** [review]
LGTM! Just a few queries.

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md:50]: Agree. Will update this.

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md [hunk 2]
```diff
@@ -96,61 +97,76 @@ Request an access token by making a request to your Okta [Org Authorization Serv
...
-5. In the dialog box that appears, enter a name for the token and select **Authorization Code (With PKCE)** as the grant type.
-6. Define the following for the token request:
+4. On the right pane, got to the  **Configure New Token** section.
+5. In the first field, enter a name for the token and select **Authorization Code (With PKCE)** as the grant type.
+6. Define the remaining fields for the token request:
 
     * **Callback URL**: Define the callback location where Okta returns the token after the user finishes authenticating. This URL must match one of the redirect URIs that you configured in the [Create an OAuth 2.0 app in Okta](#create-an-oauth-2-0-app-in-okta) section.
     * **Auth URL**: Enter the authorization endpoint for your Org Authorization Server, for example, `https://${yourOktaDomain}/oauth2/v1/authorize`.
     * **Access Token URL**: Enter the token endpoint for your Org Authorization Server, for example, `https://${yourOktaDomain}/oauth2/v1/token`.
+    * **Client ID**: Use the `client_id` of your Okta OAuth 2.0 application that you created in the [Create an OAuth 2.0 app in Okta](#create-an-oauth-2-0-app-in-okta) section.
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md:118]: I think this is more Postman-related and if a user wants to use them can follow their docs. I don't see it as critical for this demonstration exercise. (As an fyi, I did manage to create variables but it was a bit awkward, so, don't want to send users down this road.)

**Pull request merged**